src/integration/java/io/pinecone/integration/controlPlane/pod/ConfigureIndexTest.java

```diff
@@ -137,10 +143,6 @@ public void sizeIncrease() throws InterruptedException {
             });
         } catch (Exception exception) {
             throw new PineconeException("Test failed: " + exception.getLocalizedMessage());
-        } finally {
```

Comment:
I don't think `@Test` methods in a class are guaranteed to run in the same order every time, so I don't think this is guaranteed to run last which could cause issues.

Moving the clean up to `@AfterAll` helps with this.